upload-api/metrics.js

```diff
@@ -1,38 +1,28 @@
 import { hasOkReceipt } from './utils.js'
 
 import {
-  BLOB_ADD,
   STORE_ADD,
-  BLOB_REMOVE,
   STORE_REMOVE,
-  UPLOAD_ADD,
-  UPLOAD_REMOVE,
   METRICS_NAMES,
   SPACE_METRICS_NAMES,
 } from './constants.js'
 
 /**
  * @typedef {import('@ucanto/interface').Capability} Capability
- * @typedef {import('@web3-storage/upload-api').StoreRemoveSuccess} StoreRemoveSuccess
- * @typedef {import('@web3-storage/upload-api').BlobRemoveSuccess} BlobRemoveSuccess
+ * @typedef {import('@storacha/upload-api').StoreRemoveSuccess} StoreRemoveSuccess
+ * @typedef {import('@storacha/upload-api').SpaceBlobRemoveSuccess} SpaceBlobRemoveSuccess
  */
 
 /**
  * Update total admin metrics for upload-api receipts.
  * Metrics:
- * - BLOB_ADD_TOTAL: increment number of `blob/add` success receipts
- * - BLOB_ADD_SIZE_TOTAL: increment size of `blob/add` success receipts
  * - STORE_ADD_TOTAL: increment number of `store/add` success receipts
  * - STORE_ADD_SIZE_TOTAL: increment size of `store/add` success receipts
- * - UPLOAD_ADD_TOTAL: increment number of `upload/add` success receipts
- * - BLOB_REMOVE_TOTAL: increment number of `blob/remove` success receipts
- * - BLOB_REMOVE_SIZE_TOTAL: increment size of `blob/remove` success receipts
  * - STORE_REMOVE_TOTAL: increment number of `store/remove` success receipts
  * - STORE_REMOVE_SIZE_TOTAL: increment size of `store/remove` success receipts
- * - UPLOAD_REMOVE_TOTAL: increment number of `upload/remove` success receipts
```

Comment:
Only store protocol related metrics are driven by the ucan stream now. When store protocol is decommissioned the ucan stream can also be removed.